pkg/controller/unhandled.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 VMware
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controller
+
+import (
+	"errors"
+)
+
+func NewUnhandledError(err error) error {
```

Comment:
it might be worth leaving a comment that describes what "Unhandled" means in the context of `controller` given how generic the notion of "handled" can be - wdyt?